macros/plotTimeSeries.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+
+def makeInputList(chamberName, scanType):
+    ofname = getDirByAnaType(scanType,chamberName)+'listOfScanDates_gemPlotter.txt'
+    with open(template,"rt") as fin:
+        with open(ofname, "wt") as fout:
+            for line in fin:
+                fout.write(line.replace("GEMINI", chamberName)) 
+
+def makePlots(chamberName, scanType, vt1bump, elog_path):
+    call_command = 'sudo -u gempro -i gemPlotter.py --infilename='+getDirByAnaType(scanType,chamberName)+'listOfScanDates_gemPlotter.txt --anaType=scurveAna --branchName=threshold --make2D --alphaLabels -c -a'
+    os.system(call_command)
+    call_command = 'sudo -u gempro -i gemPlotter.py --infilename='+getDirByAnaType(scanType,chamberName)+'listOfScanDates_gemPlotter.txt --anaType=scurveAna --branchName=noise --make2D --alphaLabels -c -a --axisMax=25'
+    os.system(call_command)
+    call_command = 'sudo -u gempro -i gemPlotter.py --infilename='+getDirByAnaType(scanType,chamberName)+'listOfScanDates_gemPlotter.txt --anaType=scurveAna --branchName=mask --make2D --alphaLabels -c -a --axisMax=1'
+    os.system(call_command)
+    call_command = 'sudo -u gempro -i gemPlotter.py --infilename='+getDirByAnaType(scanType,chamberName)+'listOfScanDates_gemPlotter.txt --anaType=scurveAna --branchName=maskReason --make2D --alphaLabels -c -a --axisMax=32'
+    os.system(call_command)
+    call_command = 'sudo -u gempro -i gemPlotter.py --infilename='+getDirByAnaType(scanType,chamberName)+'listOfScanDates_gemPlotter.txt --anaType=scurveAna --branchName=vthr --alphaLabels -c -a'
+    os.system(call_command)
+    call_command = 'sudo -u gempro -i mkdir -p '+elog_path+'/timeSeriesPlots/'+chamberName+'/'+vt1bump+'/'
+    os.system(call_command)
+    call_command = 'sudo -u gempro -i mv '+elog_path+'/summary*.png '+elog_path+'/timeSeriesPlots/'+chamberName+'/'+vt1bump+'/'
+    os.system(call_command)
+    call_command = 'sudo -u gempro -i mv '+elog_path+'/gemPlotter*.root '+elog_path+'/timeSeriesPlots/'+chamberName+'/'+vt1bump+'/'
```

Comment:
I still don't like (know if they're necessary) these lines with the `sudo -u gempro`
* Does `plotTimeSeries`, when run as `gempro` need these lines?
  * If not, then this part should be completely agnostic to the user
  * If so, then it should be tweaked to go back up and pass the previous bullet

Reply:
I have removed these `sudo -u gempro -i` and tested at P5. Corresponding patch pushed